release/_bom.sh

```diff
@@ -383,4 +461,32 @@ function _manage_bom_jar {
 	fi
 
 	rm -fr jar-temp
+}
+
+function _manage_distro_jar {
+	lc_log DEBUG "Processing release.dxp.distro-'${_PRODUCT_VERSION}-${_BUILD_TIMESTAMP}' for api jar."
+
+	if [[ ! $(echo "${_OSGI_VERSION}" | grep "q") ]]
```

Comment:
@pgydev, where the variable `_OSGI_VERSION` is being set? Can you please clarify?